jfrog_platform/manifest.json

```
@@ -24,7 +24,7 @@
   "integration_id": "jfrog-platform",
   "assets": {
     "dashboards": {
-      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_dashboard.json"
+      "Jfrog Artifactory Overview": "assets/dashboards/jfrog_artifactory_overview.json"
```

```suggestion
      "Jfrog Artifactory Dashboard": "assets/dashboards/jfrog_artifactory_overview.json"
```
Let's only update the payload. Changing the keys here may have impact on duplicate dashboards being synced.